src/mempool/mempool.ts

```diff
@@ -349,86 +301,75 @@ export class MemoryMempool {
             const hasHigherFees = hasHigherPriorityFee && hasHigherMaxFee
 
             if (!hasHigherFees) {
-                return [false, reason]
+                const reason =
+                    conflicting.reason === "conflicting_deployment"
+                        ? "AA10 sender already constructed: A conflicting userOperation with initCode for this sender is already in the mempool"
+                        : "AA25 invalid account nonce: User operation already present in mempool"
+
+                return [false, `${reason}, bump the gas price by minimum 10%`]
```

Comment:
note you will still have to push the op that we will be removing during `findConflictingOutstanding` back to mempool again